### PR TITLE
Chore: (Docs) Updates to the Angular documentation

### PR DESCRIPTION
**Pull request by juri-sinitson**
Issue:

## What I did

Importing styles in `.storybook/preview.js` didn't work for me. 
Tried also with and without the webpack mofications proposed above. 

This instruction worked: https://github.com/storybookjs/storybook/issues/16950#issuecomment-1017085928

## How to test
1. Create an project using NX with Angular preset like this: https://nx.dev/angular-tutorial/01-create-application
1. Create a library `demo` like this `npx nx g lib demo`
1. Create a module `test` in the library `demo` like this `npx nx g m --name test --project demo`
1. Create a component `test` in the library `demo` like this: `npx nx g c --name test --project demo`
1. Generate storybook configuration in the library `demo` like this: `npx nx g @nrwl/angular:storybook-configuration demo`
1. Generate stories for the library `demo` like this: `npx nx g @nrwl/angular:stories demo`
1. Create a file `styles.scss` somewhere in the project
1. Apply the documentation changes made in this PR
1. Enjoy: `npx nx run demo:storybook`

### CODE REVIEW

**Comment by jonniebigodes** [comment]
@juri-sinitson thanks for the time and effort you've put into this pull request. We appreciate your willingness to help improve our documentation. I was looking at this,  and there's already an open pull request to address this situation (#17387). There are just a couple of items that need follow-up in that pull request and it will be merged. One thing that we could do to help complement the documentation is the following, once the pr I mentioned is merged you could make a follow-up and add the NX instructions to provide a bit more clarity? 

Let me know what you think. Would like to hear your thoughts on it.

Have a great weekend!

Stay safe

**Comment by juri-sinitson** [comment]
@jonniebigodes 

> ... once the pr I mentioned is merged you could make a follow-up and add the NX instructions to provide a bit more clarity
> ...
> Let me know what you think. Would like to hear your thoughts on it.

I think it's a good idea. Until that I'll wait until the PR you mentioned above is merged.

File: docs/configure/styling-and-css.md
Comment (ThibaudAV) [inline docs/configure/styling-and-css.md:35]: Just some information. I don't know if it is interesting. I don't even know if it's necessary to add it here. but FYI

If the user uses the `browserTarget` option without the `styles` option then it will use the styles of the target.
If both are given the style defined here will be used.

In some cases we will want to add a main scss file only for Storybook. For example if we have one Storybook for several projects and applications inside angular workspace
Ex : 
```suggestion
        "styles": [".storybook/styles.scss"]
```

**Review by ThibaudAV — APPROVED** [review]
Thank you for this work. I know that I'm not very available and active lately, but don't hesitate to ask me questions if something seems unclear

File: docs/configure/styling-and-css.md
Comment (juri-sinitson) [reply docs/configure/styling-and-css.md:35]: > If the user uses the browserTarget option without the styles option then it will use the styles of the target.
If both are given the style defined here will be used.

This does not seem to work with Nx and Angular both version 13.x. Have just tired with a production project. It looks like you really need to tell storybook directly which styles to use.

**Comment by juri-sinitson** [comment]
@jonniebigodes 
> you could make a follow-up and add the NX instructions to provide a bit more clarity

I've just made an new PR #17520 with instructions for NX.

**Comment by jonniebigodes** [comment]
@juri-sinitson appreciate the follow-up and willingness to make the pull request to address the issue. I'll check it today (Wednesday) and let you know of any additional feedback. Thank you very much 🙏 !

**Pull request closed without merging**
